test/user-test.js

```diff
@@ -74,99 +72,115 @@ describe('User', function() {
+      expect(user1.favoriteRecipes.length).to.equal(1)
+      user1.removeRecipe(recipe)
+      expect(user1.favoriteRecipes.length).to.equal(0)
+    });
   });
 
-  it('should be able to filter recipes by type in favoriteRecipes', function() {
-    user1.saveRecipe(recipe);
-    expect(user1.filterRecipes('italian', user1.favoriteRecipes)).to.deep.equal([recipe]);
-  });
+  describe('decideToCook', function() {
+    it('should be able to decide to cook a recipe', function() {
+      user1.decideToCook(recipe);
+      expect(user1.recipesToCook[0].name).to.equal('Chicken Parm');
+    });
 
-  it('should be able to filter recipes by type in recipesToCook', function() {
-    user1.decideToCook(recipe);
-    expect(user1.filterRecipes('italian', user1.recipesToCook)).to.deep.equal([recipe]);
+    it('should not add to the recipesToCook array if the recipe is undefined', function() {
+      user1.decideToCook(undefined);
+      expect(user1.recipesToCook.length).to.equal(0);
+    });
   });
 
-  it('should return null if the type is undefined in the filterRecipes function when looking in favoriteRecipes', function() {
-    let recipe1 = {name: 'strawberry shortcake biscuit', type: undefined};
-    user1.saveRecipe(recipe1);
-    expect(user1.filterRecipes(undefined,
-    user1.favoriteRecipes)).to.equal(null)
-  })
+  describe('filterRecipes', function() {
+    it('should be able to filter recipes by type in favoriteRecipes', function() {
+      user1.saveRecipe(recipe);
+      expect(user1.filterRecipes('italian', user1.favoriteRecipes)).to.deep.equal([recipe]);
+    });
 
-  it('should return null if the type is undefined in the filterRecipes function when looking in recipes to cook', function() {
-    let recipe1 = {name: 'strawberry shortcake biscuit', type: undefined};
-    user1.saveRecipe(recipe1);
-    expect(user1.filterRecipes(undefined,
-    user1.recipesToCook)).to.equal(null)
-  })
+    it('should be able to filter recipes by type in recipesToCook', function() {
+      user1.decideToCook(recipe);
+      expect(user1.filterRecipes('italian', user1.recipesToCook)).to.deep.equal([recipe]);
+    });
 
-  it('should be able to search recipes by name', function() {
-    user1.saveRecipe(recipe);
-    expect(user1.searchForRecipe('Chicken Parm')).to.deep.equal([recipe]);
+    it('should return null if the type is undefined in the filterRecipes function when looking in favoriteRecipes', function() {
+      let recipe1 = {name: 'strawberry shortcake biscuit', type: undefined};
+      user1.saveRecipe(recipe1);
+      expect(user1.filterRecipes(undefined,
+        user1.favoriteRecipes)).to.equal(null)
+      });
+
+    it('should return null if the type is undefined in the filterRecipes function when looking in recipes to cook', function() {
+      let recipe1 = {name: 'strawberry shortcake biscuit', type: undefined};
+      user1.saveRecipe(recipe1);
+      expect(user1.filterRecipes(undefined, user1.recipesToCook)).to.equal(null)
```

Comment:
I like how you grouped the two sad path tests together under the happy paths. Keeps things organized.